docs/RouteRequest.md

```diff
@@ -192,22 +192,31 @@ and in the [transferRequests in build-config.json](BuildConfiguration.md#transfe
 **Since version:** `2.0` ∙ **Type:** `duration` ∙ **Cardinality:** `Optional` ∙ **Default value:** `"PT0S"`   
 **Path:** /routingDefaults 
 
-The minimum extra time after exiting a public transport vehicle.
+The extra time after exiting a public transport vehicle.
 
 The slack is added to the time when going from the transit vehicle to the stop.
+This also influences the total time it takes to transfer. See `transferSlack` for more details about
+how the total transfer time is calculated.
+
```

Comment:
`alightSlackForMode` could be mentioned here.

Reply:
done.